spec/mailers/user_sign_in_mailer_spec.rb

```diff
@@ -1,6 +1,11 @@
 require "rails_helper"
 
 RSpec.describe UserSignInMailer, type: :mailer do
+
+  before do
```

Comment:
To explain this for posterity
See issue 1644 of https://github.com/rspec/rspec-rails/
```
routes.draw
```
combined with parallel tests causes routes to drop